xdem/examples.py

```diff
@@ -5,7 +5,7 @@
 import geopandas as gpd
 import rasterio as rio
 
-EXAMPLES_DIRECTORY = os.path.abspath(os.path.join(os.path.dirname(__file__), "examples/"))
+EXAMPLES_DIRECTORY = os.path.abspath(os.path.join(os.path.dirname(__file__), "../", "examples/"))
```

Comment:
Ok, so the unique one will be directly in the base directory?
Sounds good to me.

Reply:
@adehecq Yep!